pkg/backup/item_backupper.go

```diff
@@ -502,7 +502,7 @@ func (ib *itemBackupper) takePVSnapshot(obj runtime.Unstructured, log logrus.Fie
 			continue
 		}
 		if volumeID == "" {
-			log.Infof("No volume ID returned by volume snapshotter for persistent volume")
+			log.Error("No volume ID returned by volume snapshotter for persistent volume")
```

Comment:
This part logic is delicate. I am afraid this change may break other cases. For example, for vsphere-plugin, the volume is handled up by vsphere-plugin, but the workflow will also come here, but should not treated as errors.

Just one example, we should be careful to change the logic here unless we are sure this work for all cases.

Reply:
For vsphere-plugin, they used one tricky way to avoid calling the CreateSnapshot volumeSnapshotter interface by returning empty `volumeID` in advance and they used their own way of snapshot for volume, which is not volumeSnapshotter regular workflow. so we need one exceptional handle for the empty volumeSnapshotter when using vsphere plugin. but for other plugins, it should output an error and backup partially failed or it will be misleading for user that their backup is in complete status but volume is not been backed up.